html/pfappserver/root/static.alt/src/views/Configuration/_components/BillingTiersList.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <b-card no-body>
+    <pf-config-list
+      :config="config"
+      :isLoading="isLoading"
+    >
+      <template slot="pageHeader">
+        <b-card-header><h4 class="mb-0" v-t="'Billing Tiers'"></h4></b-card-header>
+      </template>
+      <template slot="buttonAdd">
+        <b-button variant="outline-primary" :to="{ name: 'newBillingTier' }">{{ $t('Add Billing Tier') }}</b-button>
+      </template>
+      <template slot="emptySearch">
+        <pf-empty-table :isLoading="isLoading">{{ $t('No billing tiers found') }}</pf-empty-table>
+      </template>
+      <template slot="buttons" slot-scope="item">
+        <b-button size="sm" variant="outline-primary" :to="{ name: 'TODO' }" class="float-right">{{ $t('Traffic Shaping') }}</b-button>
+      </template>
+    </pf-config-list>
+  </b-card>
+</template>
+
+<script>
+import pfConfigList from '@/components/pfConfigList'
+import pfEmptyTable from '@/components/pfEmptyTable'
+import {
+  pfConfigurationBillingTiersListColumns as columns,
+  pfConfigurationBillingTiersListFields as fields
+} from '@/globals/pfConfiguration'
+
+export default {
+  name: 'BillingTiersList',
+  components: {
+    pfConfigList,
+    pfEmptyTable
+  },
+  data () {
+    return {
+      config: {
+        columns: columns,
+        fields: fields,
+        rowClickRoute (item, index) {
+          return { name: 'billing_tier', params: { id: item.id } }
+        },
+        searchPlaceholder: this.$i18n.t('Search by name or description'),
+        searchableOptions: {
+          searchApiEndpoint: 'config/billing_tiers',
+          defaultSortKeys: ['id'],
+          defaultSearchCondition: {
+            op: 'and',
+            values: [{
+              op: 'or',
+              values: [
+                { field: 'id', op: 'contains', value: null },
+                { field: 'notes', op: 'contains', value: null }
+              ]
+            }]
+          },
+          defaultRoute: { name: 'configuration/billing_tiers' }
+        },
+        searchableQuickCondition: (quickCondition) => {
+          return {
+            op: 'and',
+            values: [
+              {
+                op: 'or',
+                values: [
+                  { field: 'id', op: 'contains', value: quickCondition },
+                  { field: 'name', op: 'contains', value: quickCondition }
```

Comment:
I think we should add `description`.

Reply:
And `notes`, list should use the same fields as `defaultSearchCondition` above.

Reply:
If we start using `notes`, we'll want to consume this field in the [billing templates](https://github.com/inverse-inc/packetfence/tree/devel/html/captive-portal/templates/billing).